docs/topics/api.rst

```diff
@@ -165,3 +165,24 @@ with a saved credit card, submit ``pay_method_uri``.
 
 .. _`payment method`: https://solitude.readthedocs.org/en/latest/topics/braintree.html#id2
 .. _`plan ID`: https://developers.braintreepayments.com/javascript+python/reference/response/plan
+
+Webhooks
+~~~~~~~~
+
+When Braintree completes certain actions, webhooks will be sent. For more information
+see the `Braintree documentation <https://developers.braintreepayments.com/javascript+python/reference/general/webhooks>`_.
+
+.. http:get:: /api/braintree/webhook/
```

Comment:
I think it would make sense to have this URL match solitude, ``/api/braintree/webhook/verify/``, and the same for ``parse``